packages/flutter/test/material/stepper_test.dart

```diff
@@ -1815,6 +1815,82 @@ testWidgets('Stepper custom indexed controls test', (WidgetTester tester) async
 
     expect(getContentClipRect().clipBehavior, equals(Clip.hardEdge));
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/160156.
+  testWidgets('Vertical stepper border displays correctly', (WidgetTester tester) async {
+    int index = 0;
+    const Color connectorColor = Color(0xff00ffff);
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: Center(
+            child: StatefulBuilder(
+              builder: (BuildContext context, StateSetter setState) {
+                return Stepper(
+                  currentStep: index,
+                  connectorColor: const WidgetStatePropertyAll<Color>(connectorColor),
+                  onStepTapped: (int value) {
+                    setState(() {
+                      index = value;
+                    });
+                  },
+                  steps: const <Step>[
+                    Step(
+                      title: Text('step1'),
+                      content: Text('step1 content'),
+                    ),
+                    Step(
+                      title: Text('step2'),
+                      content: Text('step2 content'),
+                    ),
+                  ],
+                );
+              }
+            ),
+          ),
+        ),
+      ),
+    );
+    await tester.pumpAndSettle();
+
+    final Finder findConnector = find.descendant(
+      of: find.byType(Stepper),
+      matching: find.descendant(
+        of: find.byType(PositionedDirectional),
+        matching: find.byElementPredicate((BuildContext context) {
+          if (context case BuildContext(
+            widget: ColoredBox(color: connectorColor),
+            size: Size(width: 1.0, height: > 0),
+          )) {
+            return true;
+          }
+          return false;
+        }),
+      ),
+    );
+
+    void verifyConnector() {
+      expect(findConnector, findsOneWidget);
+      final RenderBox renderBox = tester.renderObject(findConnector);
+      expect(renderBox, paints..rect(color: connectorColor));
+    }
+
+    verifyConnector();
+
+    final Finder findStep2 = find.text('step2');
+    await tester.tap(findStep2);
+
+    const int checkCount = 5;
+    final Duration duration = Duration(
+      microseconds: kThemeAnimationDuration.inMicroseconds ~/ (checkCount+1),
+    );
+
+    for (int i = 0; i < checkCount; i++) {
+      await tester.pump(duration);
+      verifyConnector();
+    }
+  });
+
```

Comment:
```suggestion
    }
  });
```

Reply:
It's weird—GitHub isn't showing that any suggestion is being made, despite how the comment is attached to a blank line.

![invalid](https://github.com/user-attachments/assets/17f503f5-5fdc-4d99-be87-07c991f5df23)

(should be fixed in https://github.com/flutter/flutter/pull/160193/commits/d70a6b5f552222336cb6e7ee42b76e33fd2b24c8)